src/features/offer/components/SameArtistPlaylist/hook/useSameArtistPlaylist.native.test.ts

```diff
@@ -0,0 +1,99 @@
+import { useSameArtistPlaylist } from 'features/offer/components/SameArtistPlaylist/hook/useSameArtistPlaylist'
+import { mockedAlgoliaOffersWithSameArtistResponse } from 'libs/algolia/__mocks__/mockedAlgoliaResponse'
+import { useNetInfoContext as useNetInfoContextDefault } from 'libs/network/NetInfoWrapper'
+import { reactQueryProviderHOC } from 'tests/reactQueryProviderHOC'
+import { renderHook, waitFor } from 'tests/utils'
+
+import * as fetchOffersByArtist from './../api/fetchOffersByArtist'
+
+jest.mock('libs/network/useNetInfo', () => jest.requireMock('@react-native-community/netinfo'))
+const mockUseNetInfoContext = useNetInfoContextDefault as jest.Mock
+
+jest.mock('libs/react-query/usePersistQuery', () => ({
+  usePersistQuery: jest.requireActual('react-query').useQuery,
+}))
+
+const fetchOffersByArtistSpy = jest
+  .spyOn(fetchOffersByArtist, 'fetchOffersByArtist')
+  .mockResolvedValue(mockedAlgoliaOffersWithSameArtistResponse)
+
+describe('useSameArtistPlaylist', () => {
+  it('should fetch same artist playlist when netInfo is connected', async () => {
```

Comment:
`should fetch same artist playlist when user has Internet connection` c'est pas plus parlant ?